components/o-tooltip/src/js/tooltip.js

```diff
@@ -123,6 +123,10 @@ class Tooltip {
 			Tooltip.throwError("tooltip.target is not set. An target for the tooltip to point at must be provided");
 		}
 
+		if (!typeof opts.target === 'string' || !opts.target instanceof HTMLElement) {
```

Comment:
I think we want this to be `&&` over `||`? Not string and not element. 

```suggestion
		if (typeof opts.target !== 'string' && !(opts.target instanceof HTMLElement)) {
```

Reply:
Ah no. Looks like the test you added was passing. This does look easier to read to my eye though, and avoids the linting error